### PR TITLE
Ajout de PALOIT

### PR DESCRIPTION
**Pull request by sboucherit**
Description de Leonard Lys, Samy Boucherit ainsi que d'Arthur Micoulet

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -16401,7 +16401,129 @@ var data = {
         mail: 'renan@bearstudio.fr',
       },
     },
+    {
+  "since": "2019-07-03",
```
Comment (dadoonet) [inline baggers.js]: Could you check the indentation?

File: baggers.js [hunk 2]
```diff
@@ -16401,7 +16401,129 @@ var data = {
...
+      ],
+    },
+  ],
+  "cities": ['Paris', 'Nancy', 'Toulouse', 'Nantes'],
+  "contacts": {
+    "mail": "llys@palo-it.com"
+  },
+},
   ],
+
```
Comment (dadoonet) [inline baggers.js]: Could you remove this change?

**Review by dadoonet — CHANGES_REQUESTED** [review]
I left some minor comments. Could you fix them?

File: baggers.js (hunk 2 above)
Comment (sboucherit) [reply baggers.js]: That should be solved ! Thank you

File: baggers.js [hunk 3]
```diff
@@ -16416,91 +16416,125 @@ var data = {
       },
     },
     {
-      since: '2015-03-10',
```
Comment (dadoonet) [inline baggers.js:16419]: I don't understand why this is removed. Could you check?
Make sure you put also your addition at the end of the file.

**Review by dadoonet — CHANGES_REQUESTED** [review]

**Pull request closed without merging**